docs/reference/api/drivers/RawSerial.md

```diff
@@ -1 +1,34 @@
 ## RawSerial
+
+The RawSerial interface provides UART functionality without the use of streams like the Serial class. This makes it suitable for use in interrupt handlers with the RTOS.
```

Comment:
Query: Does this mean the Serial class is a stream, or the Serial class provides UART functionality without the use of streams?

Reply:
The RawSerial class provides UART functionality without being a stream, the Serial class inherits from the Stream class.